config/selectionCfg_ttCR_Legacy2016.cfg

```diff
@@ -0,0 +1,175 @@
+OSinviso = isOS != 0 && ((dau1_deepTauVsJet >= 5) || (dau1_iso < 0.15) || (dau1_eleMVAiso == 1)) && dau2_deepTauVsJet >= 2 && dau2_deepTauVsJet < 5  # C region
+SSinviso = isOS == 0 && ((dau1_deepTauVsJet >= 5) || (dau1_iso < 0.15) || (dau1_eleMVAiso == 1)) && dau2_deepTauVsJet >= 2 && dau2_deepTauVsJet < 5  # D region
+
+
+# CR ttbar final category
+ttCR_invMcut = baseline , btagMM, isBoosted != 1 , invEllypsMassCut, !(isVBF == 1 && VBFjj_mass > 500 && VBFjj_deltaEta > 3 && ((bjet1_bID_deepFlavor > 0.3093) || (bjet2_bID_deepFlavor > 0.3093)))
+
+
+#########################################################################
+#########################################################################
+## weights to be applied for a certain selections when calling Fill()
+## multiple weights are passed as list and are multiplied together
+## NOTE: no weight is applied for data (the simple Fill() is used)
+[selectionWeights]
+baseline = MC_weight, PUReweight, PUjetID_SF, L1pref_weight, prescaleWeight, trigSF, IdAndIsoAndFakeSF_deep_pt, DYscale_MTT
+btagMM   = bTagweightM
+
+
+#########################################################################
+#########################################################################
+
+[sampleWeights]
+#TTfullyHad = TTtopPtreweight
+#TTfullyLep = TTtopPtreweight
+#TTsemiLep  = TTtopPtreweight
+#DY = DYLOtoNLOreweight
+
+
+#########################################################################
+#########################################################################
+
+# define alternative weights to be tested instead of the nominal one
+[systematics]
+#IdAndIsoAndFakeSF_deep_pt = tauid_pt20to25Up:idAndIsoAndFakeSF_tauid_pt20to25_up, tauid_pt20to25Down:idAndIsoAndFakeSF_tauid_pt20to25_down, tauid_pt25to30Up:idAndIsoAndFakeSF_tauid_pt25to30_up, tauid_pt25to30Down:idAndIsoAndFakeSF_tauid_pt25to30_down, tauid_pt30to35Up:idAndIsoAndFakeSF_tauid_pt30to35_up, tauid_pt30to35Down:idAndIsoAndFakeSF_tauid_pt30to35_down, tauid_pt35to40Up:idAndIsoAndFakeSF_tauid_pt35to40_up, tauid_pt35to40Down:idAndIsoAndFakeSF_tauid_pt35to40_down, tauid_pt40toInfUp:idAndIsoAndFakeSF_tauid_pt40toInf_up, tauid_pt40toInfDown:idAndIsoAndFakeSF_tauid_pt40toInf_down, etauFR_barrelUp:idAndIsoAndFakeSF_etauFR_barrel_up, etauFR_barrelDown:idAndIsoAndFakeSF_etauFR_barrel_down, etauFR_endcapUp:idAndIsoAndFakeSF_etauFR_endcap_up, etauFR_endcapDown:idAndIsoAndFakeSF_etauFR_endcap_down
+#trigSF = trigSFDM0Up:trigSF_DM0_up, trigSFDM0Down:trigSF_DM0_down, trigSFDM1Up:trigSF_DM1_up, trigSFDM1Down:trigSF_DM1_down, trigSFDM10Up:trigSF_DM10_up, trigSFDM10Down:trigSF_DM10_down, trigSFDM11Up:trigSF_DM11_up, trigSFDM11Down:trigSF_DM11_down, trigSFeleUp:trigSF_ele_up, trigSFeleDown:trigSF_ele_down, trigSFmuUp:trigSF_mu_up, trigSFmuDown:trigSF_mu_down
+#PUjetID_SF = PUjetIDSFUp:PUjetID_SF_up, PUjetIDSFDown:PUjetID_SF_down
+#bTagweightM = bTagSFMUp:bTagweightM_up, bTagSFMDown:bTagweightM_down
```

Comment:
This part is also tricky: I have included all the syst of the three channels. We should decide if we apply all of them or if we have something else in mind

Reply:
From what I understood from our conversation the contamination from other backgrounds is really small in this CR, and we want to vary only the TT normalization in this CR right? In this case it's ok to leave these lines commented.